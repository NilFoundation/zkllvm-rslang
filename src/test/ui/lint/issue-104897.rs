// error-pattern: this file contains an unclosed delimiter
// error-pattern: this file contains an unclosed delimiter
// error-pattern: this file contains an unclosed delimiter
// error-pattern: format argument must be a string literal

fn f(){(print!(รก
