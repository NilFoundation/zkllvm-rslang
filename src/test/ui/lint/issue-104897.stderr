error: this file contains an unclosed delimiter
  --> $DIR/issue-104897.rs:6:18
   |
LL | fn f(){(print!(á
   |       --      -  ^
   |       ||      |
   |       ||      unclosed delimiter
   |       |unclosed delimiter
   |       unclosed delimiter

error: this file contains an unclosed delimiter
  --> $DIR/issue-104897.rs:6:18
   |
LL | fn f(){(print!(á
   |       --      -  ^
   |       ||      |
   |       ||      unclosed delimiter
   |       |unclosed delimiter
   |       unclosed delimiter

error: this file contains an unclosed delimiter
  --> $DIR/issue-104897.rs:6:18
   |
LL | fn f(){(print!(á
   |       --      -  ^
   |       ||      |
   |       ||      unclosed delimiter
   |       |unclosed delimiter
   |       unclosed delimiter

error: format argument must be a string literal
  --> $DIR/issue-104897.rs:6:16
   |
LL | fn f(){(print!(á
   |                ^
   |
help: you might be missing a string literal to format with
   |
LL | fn f(){(print!("{}", á
   |                +++++

error: aborting due to 4 previous errors

